src/status_im/ui/screens/screens.cljs

```diff
@@ -560,7 +561,7 @@
 
    ;[Chat] New Chat
    {:name      :new-chat
-    :on-focus  [::new-chat.events/new-chat-focus]
+    :on-focus  [::contacts.events/new-chat-focus]
```

Comment:
According to new guidelines, event names should reflect their context and not be fully namespaced.

In this case, it could be `:contacts/new-chat-focus`. The `.events` seems redundant.

Reply:
done